nuxeo-layout.html

```diff
@@ -84,8 +84,34 @@
       validate: function() {
         if (this.element && ('function' === typeof this.element.validate)) {
           return this.element.validate();
+        } else {
+          // workaroud for https://github.com/PolymerElements/iron-form/issues/218, adapted from iron-form.html
+          var valid = true;
+          if (this.element) {
+            var elements = this._getValidatableElements(this.element.root);
+            for (var el, i = 0; el = elements[i], i < elements.length; i++) {
+              valid = !!el.validate() && valid;
+            }
+          }
+          return valid;
+        }
+      },
+
+      _getValidatableElements: function(parent) {
+        var nodes = Polymer.dom(parent).querySelectorAll('*');
+        var submittable = [];
+        for (var i = 0; i < nodes.length; i++) {
+          var node = nodes[i];
+          if(!node.disabled) {
```

Comment:
nit: weird formatting, missing some spaces, etc

Reply:
Indeed, adapted this chunk from `iron-form.html`, which uses different formatting. Will fix before merging. 👍